.github/workflows/tests.yml

```diff
@@ -30,12 +30,13 @@ jobs:
         # work around issues with GPy setting matplotlib backend
         echo 'backend: Agg' > matplotlibrc
         pip install .
-    # - name: Lint with flake8
-    #   run: |
-    #     # stop the build if there are Python syntax errors or undefined names
-    #     flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-    #     # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-    #     flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        # || exit 0 so we can see the flake8 issues but don't fail the stage
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics  || exit 0
```

Comment:
Let's make this one hard failure, it seems critical that the code shall at least run